msgpack-json.cabal

```diff
@@ -14,13 +14,13 @@ build-type:    Simple
 description:
   A Haskell implementation of MessagePack <http://msgpack.org/>
   .
-  This is a fork of msgpack-haskell <https://github.com/msgpack/msgpack-haskell>,
+  This is a fork of msgpack-aeson <https://github.com/msgpack/msgpack-haskell>,
```

Comment:
Are you sure? :P

Reply:
Fairly sure? What am I missing?

Reply:
Yep, that repo is a bunch of msgpack packages in one repo. We split it across multiple repos.

Reply:
My edit was too slow. :(